### PR TITLE
Feature/andy2680 caption

### PR DESCRIPTION
**Pull request by andy2680**
amazing pull request

💣 added

### CODE REVIEW

File: _posts/0000-01-08-andy2680.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![labtocat](https://octodex.github.com/images/labtocat.png)
 {: .center}
 
-Caption here
+Laboratory Cat with explosives. :bomb:
```
Comment (andy2680) [inline _posts/0000-01-08-andy2680.md:9]: 👍

**Review by andy2680 — COMMENTED** [review]
all good

**Comment by gdebrito** [comment]
you need more 🐶

**Review by gdebrito — APPROVED** [review]

**Comment by andy2680** [comment]
lets go

**Pull request merged**